password/src/Data/Password/Validate.hs

```diff
@@ -79,18 +81,31 @@ defaultPasswordPolicy = PasswordPolicy
     uppercaseChars = Just 1,
     lowercaseChars = Just 1,
     specialChars = Nothing,
-    digitChars = Just 1,
-    charSet = defaultCharSet
+    digitChars = Just 1
   }
 
+-- | Predicate which defines the characters that can be used for a password.
+newtype CharSetPredicate = CharSetPredicate
+  { getCharSetPredicate :: Char -> Bool
+  }
+
+-- | Construct 'CharSetPredicate'
+mkCharSetPredicate :: (Char -> Bool) -> CharSetPredicate
+mkCharSetPredicate = CharSetPredicate
```

Comment:
This is kind of superfluous, isn't it?
If the data constructor is exported, there's no real need to have a separate `mkCharSetPredicate` function.